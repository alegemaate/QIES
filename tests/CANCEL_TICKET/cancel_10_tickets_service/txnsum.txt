SEL 1111 0 0 **** 0
EOS 0000 0 0000 ** 0
