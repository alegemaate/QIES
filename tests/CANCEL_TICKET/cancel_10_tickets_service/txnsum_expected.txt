CAN 11111 10 00000 **** 20181004
EOS 00000 0 00000 **** 0

